### PR TITLE
Adds link back to home page at end of exporting/downloading all files…

### PR DESCRIPTION
**Pull request by ad96**
…; RDR-343

### CODE REVIEW

**Comment by ad96** [comment]
Is the appropriate place for the translation text in: config/locales/hyrax.en.yml or config/locales/rdr.en.yml?

**Comment by coblej** [comment]
@ad96 I tend to put translations we're adding for RDR in rdr.en.yml and use hyrax.en.yml mainly to override translations from Hyrax.

File: config/locales/rdr.en.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ en:
     unexpected_multiple_results: "Unexpected multiple results for %{identifier}"
     user_not_found: "User %{user_key} could not be found"
     not_found: "%{target} could not be found"
+    home_link: "Back to Research Data Repository Home"
```
Comment (coblej) [inline config/locales/rdr.en.yml]: @ad96 If you haven't already, you may want to check with product owners about this wording; e.g., whether to include the term "Repository" or not.  I think maybe we're generally not including it in user-facing text but maybe I'm wrong.

Comment (coblej) [reply config/locales/rdr.en.yml]: @ad96 Never mind ... I just saw the new home page uses Research Data Repository so I think your translation is good as is.

Comment (ad96) [reply config/locales/rdr.en.yml]: Thanks, good idea! I've added a comment on the JIRA to get their input.

**Pull request merged**